java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2647,6 +2648,22 @@ public T xlen(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Removes the specified entries by id from a stream, and returns the number of entries deleted.
+     *
+     * @see <a href="https://valkey.io/commands/xdel/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param ids An array of entry ids.
+     * @return Command Response - The number of entries removed from the stream. This number may be
+     *     less than the number of entries if <code>ids</code>, if the specified <code>ids</code>
+     *     don't exist on the stream.
```

Comment:
Should this say "in" instead of "on"?
```suggestion
     *     don't exist in the stream.
```